script.js

```diff
@@ -1,4 +1,227 @@
+  } else if (currentGameMode == "hit or stand") {
+    if (input == "hit") {
+      playerCardInHand = deck.pop();
+      playerHand.push(playerCardInHand);
+      var playerScore = scoreCounter(playerHand);
+      console.log(playerScore);
+      var computerScore = scoreCounter(computerHand);
+      console.log(computerScore);
+      var showPlayerHand = showHand(playerHand);
+      console.log(showPlayerHand);
+      var showComputerHand = showHand(computerHand);
+      console.log(showComputerHand);
+
+      var myOutputValue = "";
+      myOutputValue =
+        "Player drew: " +
+        showPlayerHand +
+        "<br>" +
+        "Player score is: " +
+        playerScore +
+        "<br>" +
+        "Computer drew " +
+        showComputerHand +
+        "<br>" +
+        "Computer score is:" +
+        computerScore;
+
+      return myOutputValue;
+    } else if (input == "stand") {
+      var playerScore = scoreCounter(playerHand);
+      console.log(playerScore);
+      var computerScore = scoreCounter(computerHand);
+      console.log(computerScore);
+      var showPlayerHand = showHand(playerHand);
+      console.log(showPlayerHand);
+      var showComputerHand = showHand(computerHand);
+      console.log(showComputerHand);
+
+      var myOutputValue = "";
+      myOutputValue =
+        getWinner(playerScore, computerScore) +
+        "Player drew: " +
+        showPlayerHand +
+        "<br>" +
+        "Player score is: " +
+        playerScore +
+        "<br>" +
+        "Computer drew " +
+        showComputerHand +
+        "<br>" +
+        "Computer score is:" +
+        computerScore;
+      currentGameMode = "draw card";
+      deck = shuffleCards(makeDeck());
+      playerHand = [];
+      computerHand = [];
+
+      return myOutputValue;
+    } else if (input != "hit" || input != "stand") {
+      myOutputValue = "Please enter 'hit' or 'stand'.";
+
+      return myOutputValue;
+    }
+  }
+};
+
+var showHand = function (hand) {
+  var i = 0;
+  var showHandResult = "";
+  while (i < hand.length) {
+    showHandResult += hand[i].name;
+
+    if (i != hand.length - 1) {
+      showHandResult += " and ";
+    }
+    i = i + 1;
+  }
+  return showHandResult;
+};
+var scoreCounter = function (hand) {
+  var i = 0;
+  var score = 0;
+  while (i < hand.length) {
+    score = score + hand[i].rank;
+    i = i + 1;
+  }
+  return score;
+};
+
+//Who wins??
+var getWinner = function (playerScore, computerScore) {
+  if (playerScore > computerScore && playerScore < 22) {
+    return `You Win!! <br>
+            Press 'Submit' to play again! <br>`;
+  } else if (playerScore > 21) {
+    return `You went busts :( <br>
+            Press 'Submit' to play again! <br>`;
+  } else if (playerScore < computerScore) {
+    return `You lost :( <br>
+            Press 'Submit' to play again! <br>`;
+  } else if ((playerScore = computerScore)) {
+    return `It's a draw! <br>
+            Press 'Submit' to play again! <br>`;
+  }
 };
```

Comment:
Good to see you are employing other functions besides `main`! This is actually the first step to answering your question about how you can make your code more concise, the ability to use functions.